lib/provider/tests/test_boku.py

```diff
@@ -38,7 +38,7 @@ def test_good(self):
     @patch('lib.boku.client.mocks', {'verify-trx-id': (500, '')})
     def test_verify_fails(self):
         self.add_seller_boku()
-        eq_(self.client.post(self.url, data=self.sample()).status_code, 200)
+        self.client.post(self.url, data=self.sample()).status_code
```

Comment:
Ah, we don't - I will remove that. This was unrelated to the md5 stuff, the code actually checks a `BokuValidation` is raised, the `eq_(....status_code, 200)` is a distraction in the test.